packages/inter-protocol/src/proposals/replaceElectorate.js

```diff
@@ -289,33 +289,87 @@ const startNewEconomicCommittee = async (
  * Starts a new Economic Committee Charter by creating an instance with the
  * provided committee specifications.
  *
- * @param {EconomyBootstrapPowers} powers - The resources and capabilities
- *   required to start the committee.
+ * @param {EconomyBootstrapPowers &
+ *   PromiseSpaceOf<{ retiredContractInstances: MapStore<string, Instance> }>} powers
+ *   - The resources and capabilities required to start the committee.
```

Comment:
I wonder if/when `retiredContractInstances` should be added in [vats/src
/core/types-ambient.d.ts](https://github.com/Agoric/agoric-sdk/blob/master/packages/vats/src/core/types-ambient.d.ts)